base/src/com/google/idea/blaze/base/sync/projectstructure/ModuleEditorImpl.java

```diff
@@ -141,29 +143,38 @@ private File getImlDirectory(BlazeImportSettings importSettings) {
     return new File(BlazeDataStorage.getProjectDataDir(importSettings), "modules");
   }
 
+  private static void removeImlFile(final VirtualFile imlFile) {
+    if (imlFile.isDirectory()) {
```

Comment:
Did you also want to check for the case where the imlFile is empty? In theory that should be a directory, but I imagine it won't hurt to be extra defensive when deleting things.

Reply:
Good idea :slightly_smiling_face: